media-video/mjpg-streamer/metadata.xml

```diff
@@ -5,6 +5,10 @@
 		<email>andrzej.pauli@gmail.com</email>
 		<name>Andrzej Pauli</name>
 	</maintainer>
+	<maintainer type="person">
+		<email>aidecoe@gentoo.org</email>
+		<name>Amadeusz Żołnowski</name>
+	</maintainer>
```

Comment:
Why are you adding him?

Reply:
He was a supporter of this ebuild and helped me maintaining it.

Reply:
yes

Reply:
removed